src/main/java/com/ftpix/sherdogparser/parsers/FighterParser.java

```diff
@@ -122,13 +122,68 @@ public Fighter parseDocument(Document doc) throws IOException {
         } catch (Exception e) {
             // no info, skipping
         }
-        // wins
+        Elements methods = doc.select(".bio_graph .graph_tag");
+        try {
+            fighter.setWinsKO(Integer.parseInt(methods.get(0).html().split(" ")[0]));
+        } catch (Exception e) {
+            // no info, skipping
+        }
+
+        try {
+            fighter.setWinsSUB(Integer.parseInt(methods.get(1).html().split(" ")[0]));
+        } catch (Exception e) {
+            // no info, skipping
+        }
+
+        try {
+            fighter.setWinsDEC(Integer.parseInt(methods.get(2).html().split(" ")[0]));
+        } catch (Exception e) {
+            // no info, skipping
+        }
+        // methods contains 6-8 values depends on "other" loss or wins methods
+        int lossKO=defineLossKO(methods);
```

Comment:
If this method changes to boolean, then it can be used directly in the if in line 146